src/legacy/core_plugins/vis_default_editor/public/components/controls/has_extended_bounds.tsx

```diff
@@ -44,7 +45,6 @@ function HasExtendedBoundsParamEditor(props: AggParamEditorProps<boolean>) {
         !props.agg.params.min_doc_count ||
         !(isType('number')(props.agg) || isType('date')(props.agg))
```

Comment:
How can I test this case?

Reply:
This was just a leftover from legacy  (take a look at diff L175 - L185) :
https://github.com/elastic/kibana/pull/34790/files#diff-88777ed0ae47452a0e9d817898c51d51
but yeah, I think this is extra since the field can't be anything than only `number` type.